README.md

```diff
@@ -461,31 +465,36 @@ To register models to a different server, specify the desired server in the `mod
 <details>
 <summary><strong><u>Full configuration reference</u></strong></summary>
 
-- `uri`: string. Required, if this section is present.  
+- `registry_uri`: string. Required, if this section is present.  
 URI of the model registry server to which to register trained models.
 
+- `model_name`: string. Required.  
+Specifies the name to use when registering the trained model to the model registry.
+
+- `model_uri`: string. Optional.  
+Specifies the URI of the model to use for batch scoring. If empty, the latest model version produced
+by the register step is used. If the register step was cleared, the latest version of the 
+registered model specified by the `model_name` attribute of the `pipeline.yaml` [register step](#register-step) 
+will be used.  
+<u>Example</u>: 
+  ```
+  model_uri: models/model.pkl
```

Comment:
I don't think `model_uri` should be specified here. This is an optional argument goes specifically with `predict` step.

Reply:
Previously this was part of the predict step config and now it is not we moved all the model related config to model_registry part. So I moved the documentation for that here as well. Is that not intended? The field is still marked as optional here.

Reply:
@kriscon-db and @jerrylian-db for FYI.
I don't think `model_uri` should be part of registry config. Sorry I missed this part when reviewing your previous PR. `model_name` however, sounds fine to me.

Reply:
Yeah! I just assumed that model_name and model_uri config changes go together along with the registry_uri . Which is why I co-located them together.
Adding @mengxr to his thoughts on this as well.